### PR TITLE
SAK-47235 Show student answer date when grading lessons questions

### PR DESCRIPTION
**Pull request by PabloArrz**
In lessons tool, when an instructor goes to grade students submissions on a lessons question, it would be nice to add the student answer date.

### CODE REVIEW

File: lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/QuestionGradingPaneProducer.java [hunk 1]
```diff
@@ -194,6 +198,7 @@ else if ("shortanswer".equals(questionType) &&
 			    decorate(new UIFreeAttributeDecorator("title", 
 								  messageLocator.getMessage("simplepage.update-points")));
 		    UIOutput.make(tofill, "grade-header",  messageLocator.getMessage("simplepage.grading-grade"));
+			UIOutput.make(tofill, "last-row-grade");
```
Comment (josecebe) [inline lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/QuestionGradingPaneProducer.java:201]: Fix tabs

File: lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/QuestionGradingPaneProducer.java [hunk 2]
```diff
@@ -208,12 +213,15 @@ else if ("shortanswer".equals(questionType) &&
 			}else {
 				UIOutput.make(branch, "student-response", user.response.getShortanswer());
 			}
+				UIOutput.make(branch, "student-date", userTimeService.dateTimeFormat(user.response.getTimeAnswered().toInstant(), null, null));
```
Comment (josecebe) [inline lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/QuestionGradingPaneProducer.java:216]: fix tabs

File: lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/QuestionGradingPaneProducer.java [hunk 3]
```diff
@@ -208,12 +213,15 @@ else if ("shortanswer".equals(questionType) &&
...
+
```
Comment (josecebe) [inline lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/QuestionGradingPaneProducer.java:217]: Remove empty line

File: lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/QuestionGradingPaneProducer.java [hunk 4]
```diff
@@ -208,12 +213,15 @@ else if ("shortanswer".equals(questionType) &&
...
-			// The grading stuff
-			UIOutput.make(branch, "student-grade");
-			UIOutput.make(branch, "gradingSpan");
-			UIOutput.make(branch, "responseId", String.valueOf(user.response.getId()));
 			if (graded) {
+				// The grading stuff
+				UIOutput.make(branch, "student-grade");
+				UIOutput.make(branch, "gradingSpan");
+				UIOutput.make(branch, "responseId", String.valueOf(user.response.getId()));
+
```
Comment (josecebe) [inline lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/QuestionGradingPaneProducer.java:224]: fix tabs

File: lessonbuilder/tool/src/resources/messages_es.properties [hunk 5]
```diff
@@ -701,6 +701,7 @@ simplepage.show-grading-pane=Mostrar panel de calificaciones
 simplepage.show-grading-pane-comments=Mostrar panel de calificaciones para comentarios
 simplepage.show-grading-pane-content=Mostrar panel de calificaciones para comentarios en el contenido de estudiante
 simplepage.grading-student=Nombre
+simplepage.grading-date=Fecha de envío
```
Comment (josecebe) [inline lessonbuilder/tool/src/resources/messages_es.properties:704]: Use unicode

**Review by josecebe — CHANGES_REQUESTED** [review]

**Comment by PabloArrz** [comment]
@josecebe I have apply the changes.

**Review by ottenhoff — APPROVED** [review]

**Comment by josecebe** [comment]
Good job @PabloArrz , LGTM!

**Review by josecebe — APPROVED** [review]

**Review by mpellicer — APPROVED** [review]
Clean, good job guys @PabloArrz and @josecebe

**Pull request merged**